_topic_maps/_topic_map.yml

```diff
@@ -1274,6 +1274,10 @@ Topics:
   Topics:
   - Name: About the OpenShift SDN network plugin
     File: about-openshift-sdn
+  - Name: Migrating to the OpenShift SDN network plugin
+    File: migrate-to-openshift-sdn
+  - Name: Rolling back to the OpenShift SDN network plugin
```

Comment:
I think this is the only misnamed spot; seems like it should be "Rolling back to the OVN-Kubernetes network plugin".